tailwind.config.js

```diff
@@ -0,0 +1,13 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: [
+    './src/pages/**/*.{ts,tsx}',
+    './src/components/**/*.{ts,tsx}',
+    './src/app/**/*.{ts,tsx}',
+    './src/stories/**/*.{ts,tsx}',
+  ],
```

Comment:
p1;

```suggestion
  content: [
    './src/pages/**/*.{ts,tsx}',
    './src/components/**/*.{ts,tsx}',
    './src/ui/**/*.{ts,tsx}',
    './src/app/**/*.{ts,tsx}',
    './src/stories/**/*.{ts,tsx}',
  ],
```

저희가 ui폴더에 컴포넌트를 작성하다보니 저부분만 추가해주시면 될 것 같습니다!